### DISCUSSION TITLE
Is there a way to disable number parsing?

### DISCUSSION THREAD

**Discussion by 50an6xy06r6n** [NaturalIntelligence/fast-xml-parser#457] [General]
I have a use-case where we're parsing an XML response that includes a zip code, and parsing those as numbers creates issues with leading zeroes on the zip code extension. Is there a way to disable number parsing entirely?

**Comment by amitguptagwl** [comment]
You can use value parser. Check for zip code format and return `null`. Check documentation for more detail

https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/docs/v4/2.XMLparseOptions.md#tagvalueprocessor
